src/StateBridge.sol

```diff
@@ -9,15 +9,22 @@ import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
 import {IWorldIDIdentityManager} from "./interfaces/IWorldIDIdentityManager.sol";
 import {ICrossDomainOwnable3} from "./interfaces/ICrossDomainOwnable3.sol";
 import {FxBaseRootTunnel} from "fx-portal/contracts/tunnel/FxBaseRootTunnel.sol";
+import "forge-std/console.sol";
 
+/// @title World ID State Bridge
+/// @author Worldcoin
+/// @notice Distributes new World ID Identity Manager roots to World ID supported networks
+/// @dev This contract lives on Ethereum mainnet and is called by the World ID Identity Manager contract
+/// in the registerIdentities method
 contract StateBridge is FxBaseRootTunnel, Ownable {
-    /// @notice boilerplate property to satisfy FxBaseRootTunnel inheritance (not going to be used)
-    bytes public latestData;
+    /*//////////////////////////////////////////////////////////////
+                                STORAGE
+    //////////////////////////////////////////////////////////////*/
```

Comment:
Please use the separator style that the identity manager does. Better to be consistent!

Reply:
I have a rust binary that I call from within VSCode that does this automatically for any word I type. What tool do you use for it?